bfdd/bfd_packet.c

```diff
@@ -1071,7 +1098,7 @@ int bp_udp_send_fp(int sd, uint8_t *data, size_t datalen,
 	uint8_t msgctl[255];
 	struct sockaddr_ll sadr_ll;
 
-
+	memset(&sadr_ll, 0, sizeof(sadr_ll));
```

Comment:
This bit could go in its own commit. Also using `= {}` initialization instead of `memset` is better due to lower overhead especially in such hot path.

https://docs.frrouting.org/projects/dev-guide/en/latest/workflow.html#defensive-coding-requirements

> For stack allocated structs and arrays that should be zero initialized, prefer initializer expressions over memset() wherever possible. This helps prevent memset() calls being missed in branches, and eliminates the error class of an incorrect size argument to memset().

Reply:
done